Libplanet.Explorer/GraphTypes/IValueType.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Bencodex;
+using GraphQL.Language.AST;
+using GraphQL.Types;
+
+namespace Libplanet.Explorer.GraphTypes
+{
+    public class IValueType : StringGraphType
+    {
+        private static readonly Codec _codec = new ();
+
+        public IValueType()
+        {
+            Name = "IValue";
+        }
+
+        public override object? Serialize(object? value)
+        {
+            if (value is Bencodex.Types.IValue iv)
+            {
+                return _codec.Encode(iv);
+            }
+
+            return value;
```

Comment:
How about throwing if unexpected types came?

You can see the `GuidGraphType` as an example.

https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL/Types/Scalars/GuidGraphType.cs

```suggestion
            return ThrowSerializationError(value);
```

Reply:
@moreal 

`ThrowSerializationError()` seems require that `value` isn't nullable. ([see also](https://github.com/graphql-dotnet/graphql-dotnet/blob/4.7.1/src/GraphQL/Types/Scalars/GuidGraphType.cs#L49-L51)).

I think that allowing null value can be fair enough because `Serialize()` has already opened the possibility. (on the other hands, we may need to care about other cases using `ThrowSerializationError()` 😓 )

Reply:
It's my mistake 😅 . Close this thread.